src/lib.rs

```diff
@@ -49,6 +49,7 @@ mod subprocess;
 mod subprocess_launcher;
 #[cfg(any(unix, feature = "dox"))]
 mod unix_socket_address;
+mod inet_address;
```

Comment:
Don't we need a `pub use inet_address::InetAddressOctet` here then?

Reply:
Ah, let me fix.